### PR TITLE
Fix failng `triangles` tests following move

### PR DESCRIPTION
**Pull request by IoannisPanagiotas**
Thank you for your contribution to the Graph Data Science Client project.

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [x] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
- [x] Your contribution is covered by tests

### CODE REVIEW

File: graphdatascience/tests/integration/test_single_mode_algos.py [hunk 1]
```diff
@@ -30,9 +32,19 @@ def run_around_tests(runner: Neo4jQueryRunner) -> Generator[None, None, None]:
...
+    result = gds.triangles(G, maxDegree=2)
+
+    assert len(result) == 1
+
+
+@pytest.mark.compatible_with(max_exlusive=ServerVersion(2, 5, 0))
+def test_alpha_triangles(gds: GraphDataScience) -> None:
+    G, _ = gds.graph.project(GRAPH_NAME, "*", {"REL": {"orientation": "UNDIRECTED"}})
+
     result = gds.alpha.triangles(G, maxDegree=2)
```
Comment (FlorentinD) [inline graphdatascience/tests/integration/test_single_mode_algos.py:48]: This one should also work for 2.5.0. we could check the deprecation is raised for 2.5.0 and later (with pytest.raise(...))

Comment (FlorentinD) [reply graphdatascience/tests/integration/test_single_mode_algos.py:48]: still an option ;)

File: graphdatascience/tests/unit/resources/example_server_endpoints.py [hunk 2]
```diff
@@ -86,7 +86,7 @@
     "gds.alpha.sllpa.write",
     "gds.alpha.sllpa.write.estimate",
     "gds.alpha.systemMonitor",
-    "gds.alpha.triangles",
+    "gds.triangles",
```
Comment (FlorentinD) [inline graphdatascience/tests/unit/resources/example_server_endpoints.py:89]: btw. these example_server_endpoints are just for a unit-test. You dont need to change them :)

**Comment by IoannisPanagiotas** [comment]
@FlorentinD 
Thanks  you for the review, I say let's merge it as is for now (when green),   I need to step out now, but I can have a look on Monday for that raise test :)  Sorry it got forgotten over fixing the tests, but i've gotten the gist of how it works now so i can tackle it i hope :)

**Comment by IoannisPanagiotas** [comment]
@FlorentinD 

i pushed a quick fix for the graph properties along iwhtu pdating the graph object for it

File: doc/sphinx/source/algorithms.rst [hunk 3]
```diff
@@ -785,6 +785,14 @@ These all assume that an object of :class:`.GraphDataScience` is available as `g
     Evaluates a division of nodes into communities based on the proportion of relationships
     that cross community boundaries.
 
+.. py:function:: gds.dag.topologicalSort.stream(G: Graph, **config: Any) -> DataFrame
```
Comment (IoannisPanagiotas) [inline doc/sphinx/source/algorithms.rst:788]: thanks for taking care of that

**Pull request merged**